runelite-client/src/main/java/net/runelite/client/plugins/banktags/tabs/BankSearch.java

```diff
@@ -77,11 +64,29 @@ public void search(InputType inputType, String search, boolean closeInput)
 			client.setVar(VarClientInt.INPUT_TYPE, inputType.getType());
 			client.setVar(VarClientStr.INPUT_TEXT, search);
 
-			client.runScript(scriptArgs);
+			layoutBank();
 		});
 	}
 
-	public void reset(boolean closeChat)
+	public void layoutBank()
+	{
+		Widget bankContainer = client.getWidget(WidgetInfo.BANK_ITEM_CONTAINER);
+		if (bankContainer == null || bankContainer.isHidden())
+		{
+			return;
+		}
+
+		Object[] scriptArgs = bankContainer.getOnInvTransmitListener();
+
+		if (scriptArgs == null)
+		{
+			return;
+		}
+
+		client.runScript(scriptArgs);
+	}
+
+	public void reset(Boolean closeChat)
```

Comment:
Why is this method signature changed to accept a boxed boolean?

Reply:
Oops, that slipped past in the rebase.